.github/workflows/docker-build.yml

```
@@ -0,0 +1,46 @@
+name: Building Docker images for PR
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build_docker:
+    name: Build docker image
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Extract PR name
+        run: echo "##[set-output name=pr;]$(echo ${{ github.ref }} | cut -d / -f 3 | tr '/' '-')"
+        id: extract_pr
+
+      - name: Build and push Docker base image
+        uses: docker/build-push-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+          repository: hellofreshtech/kangal-jmeter
+          path: docker/jmeter-base/.
+          tags: ${{ format('pr-{0}', steps.extract_pr.outputs.pr) }}
```

just to make it look the same as in other existing projects

```suggestion
          tags: ${{ format('PR-{0}', steps.extract_pr.outputs.pr) }}
```